### PR TITLE
Fix display of 'Forgot Your Password' in the checkout process

### LINKED ISSUE #27290: [FO - Responsive] - Bad display of "Forgot your password" in the checkout process
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

The link **"Forgot your password"** is bad displayed in the tab **`Personal Information>Sign`** in when your try to checkout:

image

### Expected behavior

to change the style in theme.css from:
```
body#checkout section.checkout-step .forgot-password {
    margin-left: 14.38rem;
}
```
to: 
```
body#checkout section.checkout-step .forgot-password {
    margin-bottom: 10px;
}
```
image

### Steps to reproduce

1. on mobile (width < **370px**, ex: `iPhone5s`)
2. **FO**>Click on any product
3. Click on **Add to cart** button
4. Modal will be displayed, click on **Proceed to checkout** button
5. Click on **Proceed to checkout** button again in the shopping cart page
6. Click on **Sign in** in the **PERSONAL INFORMATION** tab
7. See error

### PrestaShop version(s) where the bug happened

1.7.6.7 & 1.7.8.2

### PHP version(s) where the bug happened

7.2

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by soophoo**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | It was important to fix the display of "Forgot Your Password" in the checkout process when you are on mobile. That's why I worked on this issue in order to have a consistent display being on mobile up to a width of 425 px.
| Type?             | bug fix
| BC breaks?        |  no
| Deprecations?     | no
| Fixed ticket?     | Fixes [#27290](https://github.com/PrestaShop/PrestaShop/issues/27290)
| How to test?      | Run `npm run build` in the `_dev` directory and check The link "Forgot your password" displayed in the tab Personal Information>Sign on the checkout process page
| Possible impacts? | none

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 /vendor/
+assets
+.gitignore
```
Comment (NeOMakinG) [inline .gitignore]:
```suggestion

```

Comment (NeOMakinG) [reply .gitignore]: Meh, what's the reason behind this?

Comment (NeOMakinG) [reply .gitignore]: Don't forget the empty line too, can't add it on suggestion

File: _dev/css/components/checkout.scss [hunk 2]
```diff
@@ -440,6 +440,13 @@ body#checkout {
     .forgot-password {
       margin-left: 14.38rem;
     }
+
+    @media (max-width: 425px) {
+      .forgot-password{
+        margin-left: 0;
+        margin-bottom: 10px;
```
Comment (NeOMakinG) [inline _dev/css/components/checkout.scss]:
```suggestion
        margin-bottom: 0.625rem; 
```

**Comment by NeOMakinG** [comment]
Also, don't forget to run `npm run scss-fix`

File: .gitignore (hunk 1 above)
Comment (soophoo) [reply .gitignore]: > Meh, what's the reason behind this?

Yes sorry I'm going to fix that right now

**Review by NeOMakinG — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

File: _dev/css/components/checkout.scss [hunk 3]
```diff
@@ -440,6 +440,13 @@ body#checkout {
     .forgot-password {
       margin-left: 14.38rem;
     }
+
+    @media (max-width: 425px) {
```
Comment (NeOMakinG) [inline _dev/css/components/checkout.scss]: I'm so sorry... I missed that while reviewing...

using the `@include media-breakpoint-down(xs)` or `sm` is way better, as we use everywhere else!

**Review by NeOMakinG — CHANGES_REQUESTED** [review]

File: _dev/css/components/checkout.scss (hunk 3 above)
Comment (soophoo) [reply _dev/css/components/checkout.scss]: Yes :) I will fix that

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1 +1,3 @@
 /vendor/
+_dev/node_modules
+assets
```
Comment (Progi1984) [inline .gitignore]:
```suggestion
/_dev/node_modules
/assets
```

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Could you remove changes on `_dev/package-lock.json` ?

**Comment by HanaRebaiQA** [comment]
Hello @samuelbouye 

According to @Progi1984 ' change requests, i'll delete the `Waiting for QA` status.
And, could you please check the Conflicting files ?

Thanks!

**Review by matks — DISMISSED** [review]
@s0pho friendly reminder 😉

**Comment by eliphazbouye** [comment]
> @s0pho friendly reminder wink

Yeah sorry for the time I took to answer I going to fix the changes request

File: _dev/css/components/checkout.scss [hunk 5]
```diff
@@ -440,6 +440,13 @@ body#checkout {
     .forgot-password {
       margin-left: 14.38rem;
     }
+
+    @include media-breakpoint-down(sm) {
+      .forgot-password{
```
Comment (NeOMakinG) [inline _dev/css/components/checkout.scss]:
```suggestion
      .forgot-password {
```

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by HanaRebaiQA — COMMENTED** [review]
Hello @s0pho 

I checked this PR. The test phase is in progress.

I just check the behavior in develop branch after installing the new classic theme : 
With PR 
image

Without PR 
image


I'll give you my feedback when the test is finished.

Thanks!

**Review by HanaRebaiQA — APPROVED** [review]
Hello @eliphazed 

As confirmed by @PrestaShop/prestashop-core-developers Team, this PR will be checked only on develop branch.
So, here is my feedback : 

- Check automated tests in FO : OK
- Check with multistore : OK 
- Check with different languages : OK
- Check with different browsers (Chrome, Firefox) : OK
- Check with responsive mode : OK
- Check with mobile : OK 

Here is some screenshots : 

image
image
image
image

So, it's OK :heavy_check_mark: 

Thank you!

**Comment by Progi1984** [comment]
Thanks @eliphazed & @HanaRebaiQA

**Pull request merged**